bundle/Command/LegacySrcSymlinkCommand.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * File containing the LegacySrcSymlinkCommand class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace eZ\Bundle\EzPublishLegacyBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class LegacySrcSymlinkCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('ezpublish:legacy:symlink_src')
+            ->setDefinition(
+                array(
+                    new InputArgument('src', InputArgument::OPTIONAL, 'The src directory for legacy files', 'src/legacy_files'),
```

Comment:
How about `src/legacy` as the default?

Reply:
I was trying to not imply it should contain legacy itself, but if this is preferred or something else I'm open for it.